.github/workflows/ci.yml

```diff
@@ -8,16 +8,15 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
-    - uses: ruby/setup-ruby@v1
       with:
         bundler-cache: true
     - uses: actions/setup-node@v2-beta
       with:
-        node-version: 11
+        node-version: 15
```

Comment:
We've been standardizing on 16.7+ elsewhere. Maybe do the same here?

Reply:
👍  upgraded in 0035b88b30186296db89c9750ccbff403e69a202